Zend/zend_compile.c

```diff
@@ -2614,9 +2614,8 @@ static inline void zend_emit_assign_znode(zend_ast *var_ast, znode *value_node)
 static zend_op *zend_delayed_compile_dim(znode *result, zend_ast *ast, uint32_t type) /* {{{ */
 {
 	if (ast->attr == ZEND_DIM_ALTERNATIVE_SYNTAX) {
-		zend_error(E_DEPRECATED, "Array and string offset access syntax with curly braces is deprecated");
+		zend_error(E_COMPILE_ERROR, "Array and string offset access syntax with curly braces is not supported");
```

Comment:
```suggestion
		zend_error(E_COMPILE_ERROR, "Array and string offset access syntax with curly braces is no longer supported");
```

Maybe?

Reply:
Made the change just checking via CI I didn't forget any other place before merging